tools/build_paks.py

```diff
@@ -165,7 +165,7 @@
             "-create=" + txt_file,
             "-platform=" + platform,
             "-multiprocess",
-            "-compress"
+            "-compressed"
```

Comment:
The formatting convention I want to follow in this type of multi-line code snippet is as follows:

```
        cmd = [
            unreal_pak_bin,
            pak_file,
            "-create=" + txt_file,
            "-platform=" + platform,
            "-multiprocess",
            "-compress"
            "-compressed"]
```

I like it because it's still cleanly vertically aligned but doesn't take as much space.